Sources/FluentPostgreSQL/PostgreSQLDriver.swift

```diff
@@ -2,7 +2,9 @@ import Fluent
 import PostgreSQL
 
 public class PostgreSQLDriver: Fluent.Driver {
-    public var idKey: String = "id"
+    
+    internal static let idKey: String = "id"
+    public var idKey: String = PostgreSQLDriver.idKey
```

Comment:
Oops, I'm not sure why that would be var in the first place and not let? If it indeed needs to be var, will go with that.